### ISSUE TITLE
Import Fine-Tuned OpenAi Models

### ISSUE THREAD

**Issue by gooberstudios** [homanp/superagent#406]
### ⚠️ Please check that this feature request hasn't been suggested before.

- [X] I searched previous [Ideas in Discussions](https://github.com/homanp/superagent/discussions/categories/ideas) didn't find any similar feature requests.
- [X] I searched previous [Issues](https://github.com/homanp/superagent/labels/enhancement) didn't find any similar feature requests.

### 🔖 Feature description

openai lets you fine tune gpt 3.5 turbo and soon gpt 4, there doesnt seem to be a feature to allow you to import finetuned openai models into the super agent ui

### Acknowledgements

- [X] My issue title is concise and descriptive.
- [X] I have searched the existing issues to make sure this feature has not been requested yet.
- [X] I have provided enough information for the maintainers to understand and evaluate this request.

**Comment by homanp** [comment]
@gooberstudios I agree with this feature. Will look for a way to integrate it.
